compiler/damlc/daml-lf-conversion/src/DA/Daml/LFConversion.hs

```diff
@@ -730,21 +740,44 @@ convertBind env (name, x)
     | ConstraintTupleProjectionName _ _ <- name
     = pure []
 
+    -- Typeclass instance dictionaries
```

Comment:
This refactoring reminds me of something I did a few days ago. Is that the change I reverted?

Reply:
Yes ... I was working off the master that was reverted. This caused some merge conflicts due to the reversion, but actually having typeclass instances as a separate case in `convertBind` (as it is here) is more convenient, so the refactoring ended up sneaking back in. :)